src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -187,4 +219,23 @@ public String exec(DockerComposeExecOption options, String containerName,
             return addClusterWait(new ClusterWait(clusterHealthCheck, timeout));
         }
     }
+
+    private void removeDockerContainers(Collection<String> containerNames) throws IOException, InterruptedException {
+        Docker docker = docker();
+        try {
+            docker.rm(containerNames.toArray(new String[containerNames.size()]));
```

Comment:
I would strongly recommend overloading the rm method with both the varargs and a collections based method, having to deal with arrays when interacting with the public interface of another class tends to lead to arrays spreading throughout the codebase in my experience.